src/portfolio.js

```diff
@@ -22,6 +22,8 @@ const socialMediaLinks = {
   gmail: "saadpasta70@gmail.com",
   gitlab: "https://gitlab.com/saadpasta",
   facebook: "https://www.facebook.com/saad.pasta7"
```

Comment:
You missed a `,` here.
```suggestion
  facebook: "https://www.facebook.com/saad.pasta7",
```

Reply:
thx.  done.